src/ocean/util/container/map/Map.d

```diff
@@ -1228,6 +1228,14 @@ unittest
     test_val!(long);
 }
 
+unittest
+{
+    // ensure opIn works with const keys
+    auto map = new StandardKeyHashingMap!(int, mstring)(5);
+    auto v = "something" in map;
+    test!("is")(v, null);
+}
```

Comment:
In issue 1628, there is the following test case:
```D
void main ()
{
    StandardKeyHashingMap!(istring, istring) map;
    *map.put("Foo") = "Bar";
    cstring k = "Foo";
    assert(k in map);
}
```

I assume this will also handle it ?